### DISCUSSION TITLE
KeePassXC not opening with Mac Monterey

### DISCUSSION THREAD

**Discussion by Madhava16018** [keepassxreboot/keepassxc#7170] [Q&A]
I just upgraded my Mac to Monterey and my KeePassXC no longer opens. I get a message saying, "You can't open the application "KeePassXC" because this application is not supported on this Mac.

I've downloaded the software again and re-installed it. I've rebooted my computer, but so far it hasn't worked. This is a big problem for me as all of my passwords are on KeePass.

Any suggestions to get it running again? Is it possible to access and open the file with my passwords in some other way?

I'm not a very expert computer user and for me this is a huge, huge problem.

Thank you.

**Comment by droidmonkey** [comment]
You might have to follow portions of these directions https://support.apple.com/en-us/HT202491

**Comment by Madhava16018** [comment]
Thanks Jonathan. I already tried this, ie going to security & privacy and
clicking the open anyway button. It still didn't work. I'm getting a bit
frantic. I need my passwords.

**Reply by droidmonkey** [reply]
Those instructions are the best way to overcome these issues with Apple "blocking" software. Our application is signed AND notarized. There really should be no reason why this is happening, but it does and the only thing we can suggest is retry the instructions provided.

**Comment by Madhava16018** [comment]
Thank you Jonathan. I've tried three or four more times, (including with
rebooting). It's not working. The only difference is that now when I go to
security & privacy, there is no mention of Keepass like there was before.
I'm getting desperate. I'm locked out of my email (which is on the VMWare
Fusion part of my drive), because I don't remember my password for Windows.
In the same way there are numerous other problems of software asking me for
passwords that I don't know and have on Keepass.

Any suggestions from anyone? I really need help.

Thank you!

**Comment by droidmonkey** [comment]
Have you made sure you downloaded the correct architecture type for keepassxc? You have a choice between Apple Silicon (ARM) and Intel (x86). That choice depends on your computers hardware. You can try both just to be certain. 

As a last ditch attempt you can try Strongbox which is compatible with the keepassxc database: https://apps.apple.com/us/app/strongbox/id1270075435?mt=12

**Comment by krisgesling** [comment]
10 points for droidmonkey

https://keepassxc.org/download/ detects the Mac OS and offers you the Apple Silicon by default. So the user needs to:
1. Know the architecture of their system.
2. Not click the first big green download button they see.

The User Agent on this machine is reporting "Macintosh; Intel Mac OS X 10_15_7" ~so it should be relatively simple to address if it's a priority.~ 
(Edit: sorry not an active web dev and just learned that User Agent strings are still a mess... some ARM based Mac's still report "Intel" maybe because some devs use "ARM" to assume a mobile device 🤦 )

**Reply by droidmonkey** [reply]
Even worse, Safari actively prevents sending the architecture type of the computer in a (fruitless) attempt at preventing fingerprinting.
